crmsh/utils.py

```diff
@@ -2999,9 +3162,9 @@ def get_property(name, property_type="crm_config"):
     """
     if property_type == "crm_config":
         cib_path = os.getenv('CIB_file', constants.CIB_RAW_FILE)
-        cmd = "CIB_file={} crm configure get_property {}".format(cib_path, name)
+        cmd = "CIB_file={} sudo -E CIB_file crm configure get_property {}".format(cib_path, name)
```

Comment:
This line might has issue
```
# CIB_file=/var/lib/pacemaker/cib/cib.xml sudo -E CIB_file crm configure get_property stonith-timeout
sudo: CIB_file: command not found
```
The result is the return value of `get_property` is None

Reply:
My mistake. The parameter should be `--preserve-env=CIB_file`